stix2/utils.py

```diff
@@ -8,7 +9,13 @@
 import pytz
 import six
 
-import stix2
+import stix2.version
+
+try:
+    import stix2.registry as mappings
+except ImportError:
+    import stix2.core as mappings
```

Comment:
I tracked it down to a commit 92eeb29 on 7/13/2020 which Emmanuelle made in the stix2 generator library (affected the build_stix.py commandline script); this was when we were still working in gitlab.  It was probably at the point the package name changed, and he must have written it that way because he thought it important to preserve backward compatibility.  That got copied around and eventually wound up in the stix2 library, since I was copying generator code over.

In my mind, I was thinking someone else wrote that and must have thought it was important, so I haven't changed it.  But I can change it.

Reply:
agree we should remove it, it was for backwards compatibility. I don't think we need it anymore

Reply:
I think it originally made sense in the generator, in case someone used a previous version of stix2. I don't think we need it in stix2 itself though.